docs/android/manifest.md

```diff
@@ -0,0 +1,246 @@
+
+## Activity name
+
+The type name of an activity is based on the 64-bit cyclic redundancy check of the assembly-qualified name of the type being exported. This enables the same fully-qualified name to be provided from two different assemblies without receiving a packaging error.
+
+To override this default and explicitly specify the name of your activity, use the <xref:Android.App.ActivityAttribute.Name> property:
+
+```csharp
+using Android.App;
+
+namespace MyMauiApp;
+
+[Activity (Name="companyname.mymauiapp.activity")]
+public class MyActivity : Activity
+{
+}
+```
+
+This example produces the following XML fragment:
+
+```xml
+<activity android:name="companyname.mymauiapp.activity" />
+```
+
+> [!NOTE]
+> You should only use the `Name` property for backward-compatibility reasons, as such renaming can slow down type lookup at runtime.
+
+## Activity title bar
+
+Android app's have a title bar that displays a label. The value used for this is [`android.label`](https://developer.android.com/guide/topics/manifest/application-element.html#label), and .NET MAUI includes it in the generated manifest.
+
+To specify an activities label on the title bar, use the <xref:Android.App.ActivityAttribute.Label> property:
+
+```csharp
+using Android.App;
+
+namespace MyMauiApp;
+
+[Activity (Label="My Maui App")]
+public class MyActivity : Activity
+{
+}
+```
+
+This example produces the following XML fragment:
+
+```xml
+<activity android:label="My Maui App"
+          android:name="crc64bdb9c38958c20c7c.MainActivity" />
+```
+
+## Launch from the app chooser
+
+If your .NET MAUI Android app contains multiple activities, and you need to specify which activity should be launchable from the app launcher, use the <xref:Android.App.ActivityAttribute.MainLauncher> property:
+
+```csharp
+using Android.App;
+
+namespace MyMauiApp;
+
+[Activity (Label="My Maui App", MainLauncher = true)]
+public class MyActivity : Activity
+{
+}
+```
+
+This example produces the following XML fragment:
+
+```xml
+<activity android:label="My Maui App"
+          android:name="crc64bdb9c38958c20c7c.MainActivity">
+  <intent-filter>
+    <action android:name="android.intent.action.MAIN" />
+    <category android:name="android.intent.category.LAUNCHER" />
+  </intent-filter>
+</activity>
+```
+
+## Permissions
+
+When you add permissions to an Android app, they're recorded in the manifest file. For example, if you set the `ACCESS_NETWORK_STATE` permission, the following element is added to the manifest file:
+
+```xml
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+Debug builds automatically set the `INTERNET` permission. However, this permission is set in the generated *obj\\Debug\\net7.0-android\\AndroidManifest.xml* file. If you examine the release build version of the manifest at *obj\\Release\\net7.0-android\\AndroidManifest.xml*, the `INTERNET` permission is not present.
```

Comment:
This is true, except we put it in the MAUI project template:

https://github.com/dotnet/maui/blob/73098ab6473eb191d55ed991c474ab68d0cf493d/src/Templates/src/templates/maui-mobile/Platforms/Android/AndroidManifest.xml#L5

(Just because nearly every app uses the internet)